bundle/metadata/dependencies.yaml

```diff
@@ -3,3 +3,7 @@ dependencies:
   value:
     packageName: ocs-operator
     version: ">=4.10.0 <4.11.0 || 4.11.0"
+- type: olm.package
+  value:
+    packageName: csi-addons
+    version: "0.2.0"
```

Comment:
Don't you need the fancy `>=`, `<`, `||` pattern to ensure an upgrade to a new y stream?

Reply:
Yes, we need it downstream, Boris will change those values in the Downstream build.

Reply:
Ok

Reply:
Since we're currently not testing upgrades in our CI, this isn't a technical problem for acceptance upstream. We should fix that, of course. Later.